linode_api4/groups/monitor_service.py

```diff
@@ -0,0 +1,170 @@
+from linode_api4.errors import UnexpectedResponseError
+from linode_api4.groups import Group
+from linode_api4.objects import (
+    MonitorDashboard,
+    MonitorMetricsDefinition,
+    MonitorService,
+    MonitorServiceToken,
+)
+
+
+class MonitorGroup(Group):
+    """
+    Encapsulates Monitor-related methods of the :any:`LinodeClient`.
+
+    This group contains all features beneath the `/monitor` group in the API v4.
+    """
+
+    def list_monitor_dashboards(self, *filters) -> list[MonitorDashboard]:
+        """
+        Returns a list of dashboards.
+
+            dashboards = client.monitor_service.list_monitor_dashboards()
+            dashboard = client.load(MonitorDashboard, 1)
+
+        .. note:: This endpoint is in beta. This will only function if base_url is set to `https://api.linode.com/v4beta`.
+
+        API Documentation: https://techdocs.akamai.com/linode-api/reference/get-dashboards-all
+
+        :param filters: Any number of filters to apply to this query.
+                        See :doc:`Filtering Collections</linode_api4/objects/filtering>`
+                        for more details on filtering.
+
+        :returns: A list of Dashboards.
+        :rtype: PaginatedList of Dashboard
+        """
+
+        return self.client._get_and_filter(MonitorDashboard, *filters)
+
+    def list_dashboards_by_service(
+        self, service_type: str, *filters
+    ) -> list[MonitorDashboard]:
+        """
+        Returns a list of dashboards for a particular service.
+
+            dashboard_by_service = client.monitor_service.list_dashboards_by_service(service_type="dbaas")
+
+        .. note:: This endpoint is in beta. This will only function if base_url is set to `https://api.linode.com/v4beta`.
+
+        API Documentation: https://techdocs.akamai.com/linode-api/reference/get-dashboards
+
+        :param service_type: The service type to get dashboards for.
+        :type service_type: str
+        :param filters: Any number of filters to apply to this query.
+                        See :doc:`Filtering Collections</linode_api4/objects/filtering>`
+                        for more details on filtering.
+
+        :returns: Dashboards filtered by Service Type.
+        :rtype: PaginatedList of the Dashboards
+        """
+
+        return self.client._get_and_filter(
+            MonitorDashboard,
+            *filters,
+            endpoint=f"/monitor/services/{service_type}/dashboards",
+        )
+
+    def list_supported_services(self, *filters) -> list[MonitorService]:
```

Comment:
I'd suggest rename the function name to align with the python-sdk naming: 
```suggestion
    def services(self, *filters) -> list[MonitorService]:
```